spec/policies/presentation_policy_spec.rb

```diff
@@ -24,16 +26,56 @@
       end
     end
 
+    context 'the person is an admin' do
+      subject { described_class.new(admin, presentation) }
+      let(:admin) { FactoryGirl.build(:person, :admin) }
+      let(:presentation) { FactoryGirl.build(:presentation) }
+
+      describe '.update?' do
+        it 'returns true' do
+          expect(subject.update?).to eq(true)
+        end
+      end
+
+      describe '.edit?' do
+        it 'calls the .update? method' do
+          expect(subject).to receive(:update?)
+          subject.edit?
+        end
+      end
+    end
+
     context 'the person is a non admin' do
-      subject { PresentationPolicy.new(non_admin, 'any_object') }
+      subject { PresentationPolicy.new(non_admin, presentation) }
       let(:non_admin) { FactoryGirl.build(:person) }
+      let(:presentation) { FactoryGirl.build(:presentation) }
+
+      describe '.update?' do
+        context %(the presentation being taken action on
+        belongs to the person) do
+
+          before do
+            non_admin.presentations << presentation
+          end
 
-      policy_methods.each do |method|
-        describe method.to_s do
           it 'returns true' do
-            expect(subject.send(method)).to eq(true)
+            expect(subject.update?).to eq(true)
```

Comment:
```
  expect(obj).to be_truthy  # passes if obj is truthy (not nil or false)
  expect(obj).to be_falsey  # passes if obj is falsy (nil or false)
  expect(obj).to be_nil     # passes if obj is nil
  expect(obj).to be         # passes if obj is truthy (not nil or false)
```

I don't want to check if the value is truthy or falsey. I want to check that it is true or false.
I can do `expect(bleh).to be(true)` but it basically the exact same thing. Another point against choosing `be` or `eq` is it defies the standard we have been using outside this PR.